RHDEVDOCS-7719-new: Content creation for GitOps 1.19.6 RN (#117312)
openshift/openshift-docs: modules/gitops-release-notes-1-19-6.adoc

// Module included in the following assembly:
//
// * release_notes/gitops-release-notes-1-19.adoc

:_mod-docs-content-type: REFERENCE

[id="gitops-release-notes-1-19-6_{context}"]
= Release notes for {gitops-title} 1.19.6

[role="_abstract"]

{gitops-title} 1.19.6 is available on {OCP} 4.14, 4.16, 4.17, 4.18, 4.19, 4.20, and 4.21.

[id="errata-updates-1-19-6_{context}"]
== Errata updates

[id="RHSA-XXXX:NNNN-gitops-1-19-6-security-update-advisory_{context}"]
=== RHSA-XXXX:NNNN - {gitops-title} 1.19.6 security update advisory

Issued: 2026-08-06

> The list of security fixes that are included in this release is documented in the following advisory:

Reviewer: ```suggestion
The security fixes for this release are listed in the following advisory:
```
